From 01f05b4cbb1b0f7f400ecf7d889f3d1ec68dc007 Mon Sep 17 00:00:00 2001
From: xtex <xtexchooser@duck.com>
Date: Fri, 23 Aug 2024 17:06:41 +0800
Subject: [PATCH] FROM AOSC: remove -unknown suffix

Git tags are not available in AOSC's build environment, and -unknown suffix leads to python "packaging" module's error.
---
 autogen.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/autogen.sh b/autogen.sh
index 4e1665b98d4a..a55326dc873f 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -269,7 +269,7 @@ if [ "$myhost" = "find-version" ]; then
     else
       ingit=no
       beta=yes
-      tmp="-unknown"
+      tmp=""
       rev="0000000"
       rvd="0"
     fi

base-commit: 1a26db717575068f0ab0d00a437ae870a93e1bb8
-- 
2.46.0

